static_src/components/loading.jsx

```diff
@@ -8,6 +8,7 @@ import createStyler from '../util/create_styler';
 const LOADING_TIME = 300;
 
 const STYLES = [
+  'globalSaving',
```

Comment:
Now that we have another style, could you add a comment for each in terms of what each style do?